### PR TITLE
Integrate Tangram and OnTheRoad frameworks, and build samples for each.

### LINKED ISSUE #58: Build sample for routing

### PR DESCRIPTION
**Pull request by msmollin**
Fixes #58. This culimates the work of integrating Tangram-es and On The Road for iOS. Samples have been built out showcasing how to use each framework. Future iterations will continue to hone this approach.

### CODE REVIEW

File: Podfile.lock [hunk 1]
```diff
@@ -1,16 +1,42 @@
 PODS:
-  - Pelias (1.0.0-beta1):
-    - Pelias/Core (= 1.0.0-beta1)
-  - Pelias/Core (1.0.0-beta1)
+  - OnTheRoad (0.1)
```
Comment (ecgreb) [inline Podfile.lock]: Is the .lock file usually committed to source control?

Comment (msmollin) [inline Podfile.lock]: Yeah. It's a recommended thing these days. http://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control

File: ios-sdk/TangramVC.swift [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+//  TangramVC.swift
+//  ios-sdk
+//
+//  Created by Matt Smollinger on 7/12/16.
+//  Copyright © 2016 Mapzen. All rights reserved.
+//
+
+import UIKit
+import tangram_framework
+class TangramVC:  TGMapViewController{
```
Comment (karimnaaji) [inline ios-sdk/TangramVC.swift]: It's a little odd to mix up `TangramVC` with other names like `RoutingViewController`, they should be named following the same pattern. I personally like them explicit (leaving `ViewController`) for better understanding.

Comment (msmollin) [inline ios-sdk/TangramVC.swift]: Agreed. That's just sloppiness on my part. Will refactor in next PR.

**Pull request merged**